l10n/en/firefox/all.ftl

```diff
@@ -29,14 +26,8 @@ firefox-all-down-arrow = Choose from the list below
 
 # Used as an accessible label for a help button. The text is replaced with a "?" icon.
 firefox-all-get-help = Get help
-# Obsolete string (expires 2024-10-30)
-firefox-all-browser = Browser:
```

Comment:
**issue**:  This string should be removed in here too: https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/all/product.html#L10

Reply:
Nice catch, thanks!